multicluster/cmd/multicluster-controller/options.go

```diff
@@ -31,9 +31,7 @@ type Options struct {
 
 func newOptions() *Options {
 	return &Options{
-		// TODO: remove cert-manager dependency and
-		// use self signed cert in the future
-		SelfSignedCert: false,
+		SelfSignedCert: true,
```

Comment:
yes, the cert management is the same as antrea now since they are using the same cert_controller to do it. I may add a new section in sample setup guide later when this is merged.